.github/workflows/ci.yml

```
@@ -70,7 +77,7 @@ jobs:
 
       # ── vcpkg ─────────────────────────────────────────────────────────────
       - name: Clone vcpkg
-        run: git clone https://github.com/microsoft/vcpkg.git $VCPKG_ROOT
+        run: git clone --depth=1 https://github.com/microsoft/vcpkg.git $VCPKG_ROOT
```

actionlint (1.7.12)

[error] shellcheck reported issue in this script: SC2086:info:1:60: Double quote to prevent globbing and word splitting

(shellcheck)